### PR TITLE
Removed "experimental" from experimental_follower_read_timestamp

### LINKED ISSUE #8166: sql: drop the "experimental" in experimental_follower_read_timestamp
PR: https://github.com/cockroachdb/cockroach/pull/52359

From release notes:
> The `experimental_follower_read_timestamp()` function has been renamed to `follower_read_timestamp()`, signifying more confidence in CockroachDB's [follower read](../v20.2/follower-reads.html) implementation. The previous name remains a supported alias. [#52359][#52359] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #8166.

FYI @rmloveland

### CODE REVIEW

File: v20.2/follower-reads.md [hunk 1]
```diff
@@ -65,14 +65,14 @@ To verify that your cluster is performing follower reads:
 
 ### Run queries that use follower reads
 
-<a name="experimental-follower-read-timestamp"></a>
+<a name="follower-read-timestamp"></a>
```
Comment (rmloveland) [inline v20.2/follower-reads.md:68]: FWIW this anchor can be deleted, nothing is pointing to it (I was just making this exact PR until I saw this lol)

Comment (ericharmeling) [reply v20.2/follower-reads.md:68]: Ah! I see - sorry to step on your toes with this PR!

Comment (rmloveland) [reply v20.2/follower-reads.md:68]: You did not - in fact I'm thrilled - thanks for doing this

Comment (ericharmeling) [reply v20.2/follower-reads.md]: NP! Just removed that anchor.

**Review by andreimatei — COMMENTED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8816#-:-MLJe3R878Gm1tsExx5C:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @andreimatei and @rmloveland)

**Review by lnhsingh — COMMENTED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8816#-:-MLKpsK1AiJh3IZ0k1sF:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @lnhsingh and @rmloveland)

**Pull request merged**